deploy/examples/operator-openshift.yaml

```diff
@@ -21,7 +21,8 @@ priority:
 allowedCapabilities: ["MKNOD"]
 allowHostIPC: true
 readOnlyRootFilesystem: false
-requiredDropCapabilities: []
+# drop all default privileges
+requiredDropCapabilities: ["All"]
```

Comment:
As we are removing it because of some request, but as these are scc so it meant to remove the security permission on the container so maybe can you re-confirm it is smtg we need to drop all?

Reply:
yes, I have tested it with openshift